### ISSUE TITLE
Failing to handle Null Values: "Incorrect Type. Expected null"

### ISSUE THREAD

**Issue by timo-klarshift** [redhat-developer/vscode-yaml#277]
I have the following schema:

```
{
      "$docs": [
        "https://json-schema.org/understanding-json-schema/reference/null.html",
        "https://yaml.org/type/null.html"
      ],
      "title": "Null",
      "description": "Empty Value (null)",
      "type": "null"
    }`
```
As you can see in the screenshot it does only work for the first item where nothing is specified at all. 
As you can read here: https://json-schema.org/understanding-json-schema/reference/null.html

These Combinations should all represent null: ~, NULL, Null, null, <empty>

By looking at the source code of vscode-yaml it looks like it should be parsed correctly. I cannot find out myself whats going on. 
https://github.com/redhat-developer/vscode-yaml/blob/master/syntaxes/yaml.tmLanguage.json#L460

I am using version: 0.7.2

![2020-02-21-174356_421x328_scrot](https://user-images.githubusercontent.com/1379383/75053641-ee3b8e80-54d1-11ea-91ab-71e6dec8abd9.png)
![2020-02-21-174328_334x258_scrot](https://user-images.githubusercontent.com/1379383/75053644-ef6cbb80-54d1-11ea-9f99-81f85e0b1481.png)

Any Help appreciated. Thanks.

**Comment by timo-klarshift** [comment]
I forked the repository and created a test to verify this is not working as expected. Please see: https://github.com/redhat-developer/vscode-yaml/compare/master...timo-klarshift:issue-277-null-values-handled-incorrectly?expand=1

**Comment by timo-klarshift** [comment]
I will now also fork the language-server and see if i can verify it there as well.

**Comment by timo-klarshift** [comment]
I found the Issue in the language server: https://github.com/redhat-developer/yaml-language-server/issues/118

I forked it and created tests to verify its not working correctly. Also I have a local Fix now and will provide a pull request.

**Comment by JPinkney** [comment]
I think this can be closed as it should be in 0.8.0
